precovery/frame_db.py

```diff
@@ -505,9 +505,7 @@ def initialize_tables(self):
             sq.Column("data_offset", sq.Integer),
             sq.Column("data_length", sq.Integer),
             # Create index on midpoint mjd, healpixel, obscode
-            sq.Index(
-                "fast_query", "exposure_mjd_mid", "healpixel", "obscode", unique=True
-            ),
+            sq.Index("fast_query", "exposure_mjd_mid", "healpixel", "obscode"),
```

Comment:
I'd like @moeyensj's eyes on this - do you foresee a big impact from removing `unique=True` from this index?

Reply:
I honestly don't have a good sense of the performance impact this change will have but it seems like a necessary one to enable the upcoming pipeline. The uniqueness constraint was originally added because it was just another test to check if the input HDF5 files were sorted correctly and that exposures were correctly grouped. We assumed up until that point that we would always have the files sorted.